packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1604,6 +1647,16 @@ export class TransactionController extends BaseController<
       });
     });
   }
+
+  /**
+   * Add initial history snapshot to the provided transactionMeta history.
+   *
+   * @param transactionMeta - TransactionMeta to add initial history snapshot to.
+   */
+  private addInitialHistorySnapshot(transactionMeta: TransactionMeta) {
```

Comment:
Could this also go in `history.ts` as it seems standalone?

Reply:
Fixed in https://github.com/MetaMask/core/pull/1657/commits/577203e91035b5bb13a5b2c7151f255abe70f582